sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/cpp/CppGenerator.java

```diff
@@ -1992,7 +1993,8 @@ private CharSequence generateMessageFlyweightCode(final String className, final
             "    static const %1$s SBE_BLOCK_LENGTH = %2$s;\n" +
             "    static const %3$s SBE_TEMPLATE_ID = %4$s;\n" +
             "    static const %5$s SBE_SCHEMA_ID = %6$s;\n" +
-            "    static const %7$s SBE_SCHEMA_VERSION = %8$s;\n\n" +
+            "    static const %7$s SBE_SCHEMA_VERSION = %8$s;\n" +
+            "    static SBE_CONSTEXPR const char* SBE_SEMANTIC_VERSION = \"%13$s\";\n\n" +
```

Comment:
Why have a const expression here? Also why not have the type as `const char * const`

Reply:
I changed it to use std::string and also I removed the const expression. I added it before to be able to declare and initialize the SBE_SEMANTIC_VERSION in one single line but it was failing the msvc compiler so I changed it. Now it is declaring the constant in one line and initializing it out of the class.